### ISSUE TITLE
Change Glossary Author paging to match rest call content 

### ISSUE THREAD

**Issue by davidradl** [odpi/egeria#4078]
change Glossary Author paging to match rest call content. So a new rest call is issued every time a pagination change occurs  (change of page size, page number, next and previous page)

**Comment by davidradl** [comment]
As part of this I intend to:
- load up loading spinner between page changes. 
- get 1 more than the page size so I can set the total correctly in the pagination widget causing the right arrow to be enabled and disabled correctly 
- remove the part of the Pagination widget that shows the list of pages as this will hardly every be right and will be confusing. It relies on the total item count which we do not have.

**Comment by davidradl** [comment]
I am thinking that the client needs to know the max pagesize, so I was thinking of 
- introducing a new Subject Area API to get the maxPageSize (I think this is preferable to the client calling the admin APIs to get configuration information as that would return more than this user needs)  
- introducing a new Glossary Author view API that calls the Subject Area API to get the maxPageSize
- the Ui can then interrogate the OMVS -> OMAS get the max page size and use this knowledge if the requested page size from the user is less than the supported max page size; and can issue multiple queries to the omas to build up the page the ui user expects. 
- If the OMRS federated query finds different omrs endpoints, they could have different (lower) max page sizes, any issues in this space are out of scope for this Git issue. @grahamwallis am I right in thinking there is logic in the omrs to handle this scenario?

**Comment by davidradl** [comment]
I am now thinking that if the lower of the glossary author view and the omas maximum page size  is less than 101 (which is one more than the maximum ui page size the user can specify, so the paging widget show there is more data to get if we get 101 records back), then the OMVS should fail to start with an appropriate error.

Once this is in place we know that the javascript will not have to accumulate content with multiple rest calls to create a page to display; which is a bit tricky in out javascript rest implementation which is all promise based.
